extensions/spark/kyuubi-extension-spark-common/src/main/scala/org/apache/kyuubi/sql/KyuubiSQLConf.scala

```diff
@@ -190,4 +190,20 @@ object KyuubiSQLConf {
       .version("1.7.0")
       .booleanConf
       .createWithDefault(true)
+
+  val FINAL_WRITE_STAGE_EAGERLY_KILL_EXECUTORS_ENABLED =
+    buildConf("spark.sql.finalWriteStageEagerlyKillExecutors.enabled")
```

Comment:
it's valuable to introduce a new namespace `spark.sql.finalWriteStage.`
```suggestion
    buildConf("spark.sql.finalWriteStage.eagerlyKillExecutors.enabled")
```